addons/website/models/website.py

```diff
@@ -761,6 +762,9 @@ def get_unique_key(self, string, template_module=False):
         key_copy = string
         inc = 0
         domain_static = self.get_current_website().website_domain()
+        website_id = self.env.context.get('website_id', False)
+        if website_id:
+            domain_static = [('website_id', 'in', (False, website_id))]
```

Comment:
Useless because `get_current_website` already do that exactly

Reply:
Same as [**]